src/toil_scripts/lib/programs.py

```diff
@@ -1,35 +1,10 @@
 import os
 import subprocess
 import logging
+from bd2k.util.exceptions import panic
 
 _log = logging.getLogger(__name__)
 
-# FIXME: replace with bd2k.util.processes.which
```

Comment:
BPL's which is a generator, so the old invocations  `which('…')` have to be replaced with `next(which('…'),None)`.